client/src/App.tsx

```diff
@@ -32,6 +33,7 @@ function App(): JSX.Element {
                 <ProtectedRoute exact path="/dashboard" component={Dashboard} />
                 <ProtectedRoute path="/profile/settings" component={Settings} />
                 <ProtectedRoute path="/bookings" component={Bookings} />
+                <ProtectedRoute path="/profile/exampleid" component={ProfileDetails} />
```

Comment:
```suggestion
                <ProtectedRoute path="/profile/:example-id" component={ProfileDetails} />
```
We can pass the sitters' profile id as a url param from the listings to the profile details page

Reply:
I believe I changed this in the integration, but I suppose there is no reason to not make this change.